.env

```
@@ -2,3 +2,4 @@ AUTH_URL=https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
 REGISTRATION_URL=https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
 PROXY_URL=https://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
 PROXY_WEBSOCKET_URL=wss://konflux-ui.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/
+LIGHTSPEED_URL=http://127.0.0.1:8081
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)